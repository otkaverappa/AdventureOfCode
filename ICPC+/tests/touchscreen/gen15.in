19
vythbmalnmpunns 10
ksgtqqpafmraitc
doawxclxpalfuas
nlmteelteqhysqr
vvhvsezfdpfpwap
opackggvcnojlgb
cdxlbrpawsrdjnd
zdubhekqcmdqxqy
zscwddpuflmzowo
uxuoabvkrxywpwm
pvgrtjwifevsflo
thnpynrtaksyhhu 10
vwrcqknoutsrlzf
btmqkkdbfdoxeve
yqdqutcihzbcqob
xrwkhjwkkcqybuv
guolkiepsnovrgj
veatqhcmunqknsg
laazlnjrceeqzyy
jtclmstphpexbtr
herjfewrvavcswa
rfttrhomyvbeuey
mledwmktfftchmy 10
hfgcyxmolkjopes
pqftvbhfwonqqwe
pejvijuuyhhjvwp
plhugdxqkuezmvw
qmcbjkmdhkmowjm
lyxtufyrxknckzz
gplksuwfzdrltnw
fzueuqmsjjcxlow
kwoxggtdnsgffcu
bhtxonnahzjkwxa
ujxihdragevplct 10
ggabdortqyuakqx
lkgkvpnoswunngr
ioxipdziyrhsrti
qguzqrqgfieavti
nbwmllpmwnedfxy
pofkpxdffjqjjne
rafpprcedysjbzh
cpxjbmgetopjzax
etzliqelvkjptno
apfxyilgphueswf
raytnhjsuecdvvq 10
myfrxfcjorskvkg
ccidvwkeoglsjgp
asnhmlpowdfoncy
ufcekxaweqcpklx
anqnxeamswpzkcb
lyindsnforyqiie
firxyobyatvrshu
wuugizzxhpqfiyp
mfaddyugyuavmuf
iszezjgagnpysxw
jlejpjjjpifrftm 10
kdgkifvohcdxavw
zgifqzozkdjpvrl
jbopnyxloenrdpo
aqvkxnjlotrykor
vxsmoikltbqistz
jvsefptqdhjuhui
yrhtdvdqhwrzgls
huqbavrtmwbxsit
ajljcqefgndxmlj
fudvwgspzglcedl
xfwkqybufhikhux 10
qbrvynbtcdzpfdu
sdbonroomvywffq
cxuwusjynobnfis
abvbrlvibhdbflg
vpfrlamxyznzmsh
hskeweprmtapufc
bcsiufigeghrhwk
rdcbhafzrtsrlmy
npbhyxpgftpmlwi
vnnzrvayutsonda
lqsmaqlpyqipfvm 10
nqbdqsastwllkyq
mmieaiwmzvcikjf
yxwccowcipytciu
sugbzilxuluzvfi
afhyhjmenwvlsyv
mqqurrcfozqjylq
inxqmhbalpxgcpe
ydxozqhdvxeliey
anmyfyfiasxxaao
hatgoswxysucffh
efwsdbsklsejshl 10
gqlcwbwsaukuzqz
iwheomhgxsyddqm
qzcddygzrgwddxv
cfrllfyunxomase
nkfrokqwkjdimgf
ikkbwxjurwrfktz
rikyaajqwvzbdnh
ixuubsrkoiihnva
nolzmlzxdxsezxt
ifrebswsemamtqh
wfxhekufjxceddb 10
wmjpqkhpdlbfaxx
jtehaitunctsgyv
kujtkcftrkfushr
pqkuxmesiuvdmdb
hnwtiivobmyiiqp
aiimchbiajcxnoa
qyqorawmoaapikf
aknkzrraarleqat
trtjhmleaagapom
wqxmayfuahlnmeo
fxhaiomtumvaomr 10
ajjzxkzcezkrnwv
ddukfearyvfvxti
ovryvokusqwchja
fmfbykgybgvibtb
jjzcjustqllmpvx
sclxdjilklrfuta
xemwhvszrjmcxcx
xuzitfubqgpkllf
okjahqvbsomerki
qjcsrwxnzptqbhb
gvoqxviuwajjhcv 10
pseukxsjlrbgkcn
njkbzhyjevkngrp
dijjcwguftoicyl
pnwcpxlngpkqesj
vvsfeubnriifrkf
ecsaehxpxdhhwla
fhxzoewptnzbsso
xwqpwwztnwzudvi
fcrfcfjywcnvffo
temjvjhuevqfpwa
xdfqihvrhtvurbb 10
huftgccpxjkpoal
rzpwqzfnrohoiap
lhmsauufjrouhew
uxxltnladeomsym
kjtwbvqwabpqvyv
rvuriogtojzfxud
jenecqauocvette
qmzkdiakdrvfwtz
zefoljfldtpazit
eygfllnlvtcryam
xaqeqbnxoqteqso 10
mynueaftldmqukq
gjsynlaciqubulw
jzuytaaaundgdxt
wfcqgsbiukypnvc
jgbeewgewatajxz
cvhennwpvrbvioq
kywnecjkifnchwb
hyyoeldbaaudves
oosmbypkkzrxbzv
egtcwyobbobxsyb
morgbseseqrxotw 10
jxefacfodiegfyh
ikvzqytwqxniudd
toaxtccashixqpx
xxiuyatrzlqmtnp
wgfzfzdhbwrkvhb
sebcazfvrehjqcw
hzcmakndrpbkbwr
crvdtyfatweahwd
edegsgsfjkwnuxj
occlhyjmydkdfsz
kwfoexuxegjbtfa 10
etejfmhoyhtjkzd
mjztafswcwcnzxt
ceogntuuktdfeqg
hcsixsnpqsntfps
arwpzlkuhwnlctb
kafckcuxrlpggxv
axozookbisxxdbs
emujoenldgyvmes
igryfgmqhwkgvni
ptujcknpvtvtojz
itgzsnhgfqfpzac 10
hqvdbzprqlkngyz
iisqhmdrtjjyyia
drsauucmluzxkfx
jnfbfoojfjuphux
jzrcanyezlyyikg
fttmxbankgwgxfa
woanqcbogazgbjr
hqmccjddwnlstkz
wgowugaxvizwoai
hhywkbfngedtyyd
yujmsfsscpaeore 10
waouwyxendiswgt
benlsisncxcfbqy
hmyvjxvgblmldkt
wmamzswrfyokdpa
bxpcuazqicbupgh
ifvirubnnincsrt
vusmwnmxfwbirqo
aztxjmrmbgvpkpi
gmeyadnoculecew
texmdhbwtccqtni
dtwitxnjlpdyvic 10
sbipplyqhutmkoz
usurcpqpydgecco
eupplecmvmgqyse
znytgakwrlzypbc
diyuzjybxvndllw
sklsfrtqnmdomsq
oxamtzwtcvpqzcd
xwqkpxbipqwsejm
wxlykeziycdpuer
zdopndponegjxnu
