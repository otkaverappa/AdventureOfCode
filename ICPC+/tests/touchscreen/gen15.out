opackggvcnojlgb 53
pvgrtjwifevsflo 61
cdxlbrpawsrdjnd 62
uxuoabvkrxywpwm 65
vvhvsezfdpfpwap 65
doawxclxpalfuas 66
zdubhekqcmdqxqy 68
nlmteelteqhysqr 69
ksgtqqpafmraitc 70
zscwddpuflmzowo 76
guolkiepsnovrgj 44
jtclmstphpexbtr 47
rfttrhomyvbeuey 48
xrwkhjwkkcqybuv 50
veatqhcmunqknsg 54
btmqkkdbfdoxeve 57
herjfewrvavcswa 60
laazlnjrceeqzyy 60
vwrcqknoutsrlzf 63
yqdqutcihzbcqob 66
kwoxggtdnsgffcu 47
gplksuwfzdrltnw 49
lyxtufyrxknckzz 51
plhugdxqkuezmvw 55
bhtxonnahzjkwxa 56
pejvijuuyhhjvwp 56
qmcbjkmdhkmowjm 56
fzueuqmsjjcxlow 59
pqftvbhfwonqqwe 61
hfgcyxmolkjopes 62
pofkpxdffjqjjne 44
rafpprcedysjbzh 48
etzliqelvkjptno 49
ioxipdziyrhsrti 50
lkgkvpnoswunngr 56
cpxjbmgetopjzax 60
ggabdortqyuakqx 60
nbwmllpmwnedfxy 60
qguzqrqgfieavti 71
apfxyilgphueswf 73
asnhmlpowdfoncy 48
mfaddyugyuavmuf 48
ccidvwkeoglsjgp 49
iszezjgagnpysxw 50
myfrxfcjorskvkg 51
firxyobyatvrshu 54
lyindsnforyqiie 54
ufcekxaweqcpklx 56
wuugizzxhpqfiyp 62
anqnxeamswpzkcb 69
vxsmoikltbqistz 47
jbopnyxloenrdpo 50
aqvkxnjlotrykor 53
kdgkifvohcdxavw 53
huqbavrtmwbxsit 56
jvsefptqdhjuhui 58
fudvwgspzglcedl 61
ajljcqefgndxmlj 64
yrhtdvdqhwrzgls 67
zgifqzozkdjpvrl 68
abvbrlvibhdbflg 42
bcsiufigeghrhwk 47
cxuwusjynobnfis 50
sdbonroomvywffq 51
qbrvynbtcdzpfdu 52
vnnzrvayutsonda 55
hskeweprmtapufc 58
rdcbhafzrtsrlmy 60
npbhyxpgftpmlwi 61
vpfrlamxyznzmsh 75
ydxozqhdvxeliey 49
hatgoswxysucffh 55
nqbdqsastwllkyq 55
afhyhjmenwvlsyv 61
mqqurrcfozqjylq 61
sugbzilxuluzvfi 63
yxwccowcipytciu 68
anmyfyfiasxxaao 76
mmieaiwmzvcikjf 77
inxqmhbalpxgcpe 89
cfrllfyunxomase 50
qzcddygzrgwddxv 50
ifrebswsemamtqh 53
iwheomhgxsyddqm 56
nkfrokqwkjdimgf 56
ixuubsrkoiihnva 59
ikkbwxjurwrfktz 60
rikyaajqwvzbdnh 61
gqlcwbwsaukuzqz 64
nolzmlzxdxsezxt 66
jtehaitunctsgyv 37
pqkuxmesiuvdmdb 48
wmjpqkhpdlbfaxx 49
aknkzrraarleqat 51
trtjhmleaagapom 52
wqxmayfuahlnmeo 55
qyqorawmoaapikf 56
aiimchbiajcxnoa 61
kujtkcftrkfushr 61
hnwtiivobmyiiqp 63
sclxdjilklrfuta 45
fmfbykgybgvibtb 48
ddukfearyvfvxti 57
okjahqvbsomerki 57
ovryvokusqwchja 57
jjzcjustqllmpvx 58
xemwhvszrjmcxcx 59
qjcsrwxnzptqbhb 62
ajjzxkzcezkrnwv 64
xuzitfubqgpkllf 66
fcrfcfjywcnvffo 36
njkbzhyjevkngrp 41
temjvjhuevqfpwa 51
ecsaehxpxdhhwla 52
dijjcwguftoicyl 55
vvsfeubnriifrkf 55
pseukxsjlrbgkcn 59
fhxzoewptnzbsso 68
pnwcpxlngpkqesj 68
xwqpwwztnwzudvi 68
eygfllnlvtcryam 41
rvuriogtojzfxud 45
zefoljfldtpazit 50
jenecqauocvette 51
lhmsauufjrouhew 54
qmzkdiakdrvfwtz 54
kjtwbvqwabpqvyv 56
uxxltnladeomsym 57
huftgccpxjkpoal 60
rzpwqzfnrohoiap 61
kywnecjkifnchwb 55
jzuytaaaundgdxt 56
egtcwyobbobxsyb 57
jgbeewgewatajxz 57
gjsynlaciqubulw 61
hyyoeldbaaudves 64
wfcqgsbiukypnvc 65
mynueaftldmqukq 67
oosmbypkkzrxbzv 71
cvhennwpvrbvioq 73
crvdtyfatweahwd 38
toaxtccashixqpx 46
ikvzqytwqxniudd 50
wgfzfzdhbwrkvhb 53
jxefacfodiegfyh 54
sebcazfvrehjqcw 54
edegsgsfjkwnuxj 57
occlhyjmydkdfsz 63
hzcmakndrpbkbwr 66
xxiuyatrzlqmtnp 66
kafckcuxrlpggxv 35
hcsixsnpqsntfps 44
igryfgmqhwkgvni 44
emujoenldgyvmes 51
etejfmhoyhtjkzd 52
ptujcknpvtvtojz 53
mjztafswcwcnzxt 56
ceogntuuktdfeqg 58
arwpzlkuhwnlctb 60
axozookbisxxdbs 68
hhywkbfngedtyyd 44
fttmxbankgwgxfa 46
jnfbfoojfjuphux 48
iisqhmdrtjjyyia 52
jzrcanyezlyyikg 56
woanqcbogazgbjr 56
hqmccjddwnlstkz 60
drsauucmluzxkfx 64
hqvdbzprqlkngyz 64
wgowugaxvizwoai 69
gmeyadnoculecew 48
benlsisncxcfbqy 49
texmdhbwtccqtni 50
ifvirubnnincsrt 52
hmyvjxvgblmldkt 54
vusmwnmxfwbirqo 54
waouwyxendiswgt 55
wmamzswrfyokdpa 57
bxpcuazqicbupgh 63
aztxjmrmbgvpkpi 72
xwqkpxbipqwsejm 42
znytgakwrlzypbc 49
sbipplyqhutmkoz 53
eupplecmvmgqyse 56
zdopndponegjxnu 56
diyuzjybxvndllw 60
wxlykeziycdpuer 61
sklsfrtqnmdomsq 62
oxamtzwtcvpqzcd 66
usurcpqpydgecco 76
